### DISCUSSION TITLE
How to interstitial 90s show one time?

### DISCUSSION THREAD

**Discussion by lieuavlq** [admob-plus/admob-plus#432] [Q&A] [answered]
Please help interstitial 90s show one time 
```
await interstitial.load()
await interstitial.show()
```

**Comment by ratson** [comment, marked as answer]
@lieuavlq Is that you can not show the ad twice?
It is too little code and description about your problem.

**Reply by lieuavlq** [reply]
Ah no, it mean I want to show each 90s Ads show 1 time,  my code same in document!

**Reply by ratson** [reply]
@lieuavlq You can do it with `setTimeout`, e.g.
```js
document.addEventListener('admob.ad.dismiss', async () => {
  await interstitial.load()
  setTimeout(() => interstitial.show(), 1000 * 90)
})
```

**Reply by lieuavlq** [reply]
Thanks so much! this plugin so good! :D
